### ISSUE TITLE
no-duplicate-spread-property: false positive with Getter and Setter

### ISSUE THREAD

**Issue by ajafff** [fimbullinter/wotan#374]
```ts
({
    get foo() { return 1; },
//      ~~~                  [error no-duplicate-spread-property: Property 'foo' is overridden later.]
    set foo(v: number) {},
    ...{
        bar: 1,
    },
});
```
